test-helpers/a11y-helper.js

```diff
@@ -0,0 +1,40 @@
+/*eslint no-console: 0*/
+/*eslint no-undef: 0*/
+import 'axe-core/axe.min.js';
+
+/*
+* Rules can be disabled using the options parameter:
+* options = { "rules":
+*   {
+*     "color-contrast": { enabled: false },
+*     "valid-lang": { enabled: false }
+*   }
+* }
+* Rule ids can be found here: https://github.com/dequelabs/axe-core/blob/master/doc/rule-descriptions.md
+*/
+export async function runAxe(element, options) {
+	const results = await axe.run(element, options || {});
+
+	const violations = results.violations;
+
+	console.log('Inapplicable Tests:', results.inapplicable.length);
+	console.log('Passed Tests:', results.passes.length);
+	console.log('Failed Tests:', results.violations.length);
+
+	if (!violations.length) {
+		return Promise.resolve();
```

Comment:
Super nit: since you're returning here you could remove the `else` and reduce nesting.